testing/mustcert/example_test.go

```diff
@@ -0,0 +1,73 @@
+package mustcert
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+)
+
+// This example uses mustcert to create certificates, start a TLS server, and a client to
+// talk to it.
+func Example() {
+	ca := CA("root", nil)
+	serverCert := Leaf("localhost", ca)
+	clientCert := Leaf("client", ca)
+
+	// Create the TLS Test Server
+	server := httptest.NewUnstartedServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if _, err := w.Write([]byte("hello, world!")); err != nil {
+			fmt.Println(err)
+		}
+	}))
+
+	rootCAs := Pool(ca.TLS())
+	server.TLS = &tls.Config{
+		ClientAuth:   tls.RequireAndVerifyClientCert,
+		Certificates: []tls.Certificate{*serverCert.TLS()},
+		RootCAs:      rootCAs,
+		ClientCAs:    rootCAs,
+	}
+	server.StartTLS()
+	defer server.Close()
+
+	// Create the Client configuration
+	cert, err := tls.X509KeyPair([]byte(clientCert.CertPEM()), []byte(clientCert.KeyPEM()))
+	if err != nil {
+		fmt.Println(err)
+	}
+	caCertPool := x509.NewCertPool()
```

Comment:
are `Pool(ca.TLS())` and `caCertPool.AppendCertsFromPEM([]byte(ca.CertPEM()))` analogous?
if so, would keep the same style to not confuse the reader as to why is one different to another

Reply:
good call!